### PR TITLE
creating basic angular ssr project

### PR DESCRIPTION
**Pull request by gautamkrishnar**
Some of the SSR optimisations are still pending. Will do that once the project has released its initial version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
+  "name": "ng-universal-demo",
```
Comment (riginoommen) [inline package.json]: Update project name

File: package.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  "version": "0.0.0",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/angular/universal-starter.git"
```
Comment (riginoommen) [inline package.json]: update project repo

File: package.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  },
+  "contributors": [
```
Comment (riginoommen) [inline package.json]: update contributors

**Comment by riginoommen** [comment]
LNTM

File: package.json (hunk 1 above)
Comment (gautamkrishnar) [reply package.json]: done

**Comment by gautamkrishnar** [comment]
@riginoommen this project was actually created from one of a the best ssr starting pont https://github.com/angular/universal-starter repo. I am recommending this because since the project is quite large it may  be hard to add this feature from the starting point of the project itself. So we need not have to take extra effort on this. All files under the app folder is same as the normal angular project that we get when doing `ng new mozilla-kerala-home` all the ng commands work pretty well with this.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Mozilla-Kerala",
+  "description": "Mozilla Kerala Website",
+  "version": "0.0.1",
+  "license": "SEE LICENSE IN LICENSE",
```
Comment (riginoommen) [inline package.json]: Mention Licence Properly

File: package.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    "lint": "ng lint ng-universal-demo",
+    "build:client-and-server-bundles": "ng build --prod && ng run ng-universal-demo:server:production",
+    "build:prerender": "npm run build:client-and-server-bundles && npm run compile:server && npm run generate:prerender",
+    "build:ssr": "npm run build:client-and-server-bundles && npm run compile:server",
+    "compile:server": "tsc -p server.tsconfig.json",
+    "generate:prerender": "cd dist && node prerender",
+    "serve:prerender": "cd dist/browser && http-server",
+    "serve:ssr": "node dist/server"
+  },
+  "dependencies": {
```
Comment (riginoommen) [inline package.json]: can we create a new structure with angular 7 dependencies this looks for angular 6

File: server.ts [hunk 6]
```diff
@@ -0,0 +1,51 @@
+import 'zone.js/dist/zone-node';
```
Comment (riginoommen) [inline server.ts]: this structure loos like an ejected form of the application

File: package.json (hunk 5 above)
Comment (gautamkrishnar) [reply package.json]: Will update with angular 7

**Comment by riginoommen** [comment]
> @riginoommen this project was actually created from one of a the best ssr starting pont https://github.com/angular/universal-starter repo. I am recommending this since the project is quite large it may be hard to add this feature in a later point. Adding this from the starting point of the project itself is good so that we need not have to take extra effort on this. All files under the app folder is same as the normal angular project that we get when doing `ng new mozilla-kerala-home` all the ng commands work pretty well with this.

Hi Gautam

I checked the stater skeleton here is my observations

* Dependencies are not updated with the angular 7
* Structure seems to be the ejected mode of the angular 6 app
* in angular apps the `express` is used in a different way
* Better create the app initial structure in the official way. from there we can customize it.
`For Big Projects also we do the same.`

**Comment by gautamkrishnar** [comment]
Okay then we can close this one 👍 Lets do ssr later.

**Comment by riginoommen** [comment]
@gautamkrishnar Can you please create an angular 7 starter project from scratch

**Comment by gautamkrishnar** [comment]
@riginoommen yes doing it now

**Pull request closed without merging**